src/keymap.js

```diff
@@ -17,6 +17,12 @@ const keymap = {
     NAV_UP: 'k',
     SELECT: 'x',
     JUMP_INTO: 'enter'
+  },
+  SortedTable: {
+    SEARCH: '/',
+    NAV_DOWN: 'j',
+    NAV_UP: 'k',
+    SELECT: 'x'
```

Comment:
Please add `ROW_ACTION: 'enter'` while you're at it, which should trigger `rowAction` or `rowLink` if any.